com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/antlr4/com/ca/lsp/core/cobol/parser/CobolParser.g4

```diff
@@ -959,6 +959,15 @@ dataDescriptionEntryFormat3
 dataDescriptionEntryExecSql
    : EXECSQLLINE+ DOT_FS?
    ;
+
+//TODO: make it beautiful
```

Comment:
remove TODO or make it beautiful 💅

Reply:
I changed the logic a little bit, now I check the terminals if they have the copybook enter/exit tags. Probably, we may delete those rules